### PR TITLE
Rdiscrowd 2324

### PR DESCRIPTION
**Pull request by pwills11**
*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

File: templates/projects/tasks.html [hunk 1]
```diff
@@ -21,6 +21,12 @@
             {{ render_project_card_option(project, upload_method, title=_('Export Tasks'), explanation=_('Export tasks to JSON and CSV'), link=url_for('project.export_to', short_name=project.short_name), link_action_text=_('Export'), icon='cloud-download')}}
         </div>
         {% endif %}
+        <!-- Project owner, subadmins that are project co-owners and admins can delete tasks to project -->
+        {% if  (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
```
Comment (ywrsusan) [inline templates/projects/tasks.html:25]: Project owner should be able to delete tasks without a subadmin privilege, right?

Comment (dchhabda) [reply templates/projects/tasks.html:25]: A regular user can also be the project owner. hence only owners submin access are entitled to delete tasks.

Comment (pwills11) [reply templates/projects/tasks.html:25]: I felt like the permissions would be similar to the use case of importing tasks on line 13

Comment (ywrsusan) [reply templates/projects/tasks.html:25]: ok

**Review by ywrsusan — APPROVED** [review]
lgtm

**Pull request merged**